src/test/java/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStepTest.java

```diff
@@ -365,7 +365,7 @@ private static class Decorator extends LauncherDecorator implements Serializable
         FlowGraphTable t = new FlowGraphTable(b.getExecution());
         t.build();
         for (Row r : t.getRows()) {
-            if (r.getDisplayName().equals("Step with label")) {
+            if (r.getDisplayName().contains("Step with label")) {
```

Comment:
What is the new `Row.displayName`? `sh (Step with label)` or similar?

Reply:
correct, it's `sh (Step with label)`

Reply:
⇒ #253